CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## [10.0.1](https://github.com/theforeman/puppet-foreman_proxy_content/tree/10.0.1) (2020-06-05)
+
+[Full Changelog](https://github.com/theforeman/puppet-foreman_proxy_content/compare/10.0.0...10.0.1)
+
+**Fixed bugs:**
+
+- Fixes [\#29660](https://projects.theforeman.org/issues/29660) - Serve Pulpcore ISO via HTTPS [\#260](https://github.com/theforeman/puppet-foreman_proxy_content/pull/260) ([ekohl](https://github.com/ekohl))
```

Comment:
https://github.com/theforeman/puppet-foreman_proxy_content/pull/260 is labeled as an enhancement on the PR, but here it's listed as a bugfix.

I could see the argument for taking either approach, but I think we should be consistent.